examples/p5js/UniversalSentenceEncoder/UniversalSentenceEncoder_WithTokenizer/sketch.js

```diff
@@ -1,37 +1,44 @@
 let sentenceEncoder;
-const sentence = 'Monday, Tuesday, Wednesday, Thursday, and Friday are days of the Week';
-
+let textInput;
 
 function setup(){
   createCanvas(512, 512);
-  // background(220);
   colorMode(HSB, 360, 100, 100);
   rectMode(CENTER);
-  textAlign(CENTER);
-  sentenceEncoder = ml5.universalSentenceEncoder({withTokenizer:true}, modelLoaded)
+  textInput = createInput('Monday, Tuesday, Wednesday, Thursday, and Friday are days of the week.');
+  textInput.size(500);
+  sentenceEncoder = ml5.universalSentenceEncoder({withTokenizer:false}, modelLoaded);
 }
 
 function modelLoaded(){
-  console.log('model ready')
-  predict();
+  console.log('model ready');
+  // encode the current input.
+  encode();
+  // attach a listener to encode when the input is changed.
+  textInput.input(encode);
 }
 
-function predict(){
-  console.log('predicting')
+function encode(){
+  console.log('encoding');
+  const sentence = textInput.value();
   sentenceEncoder.encode(sentence, gotResults);
 }
 
 function gotResults(err, result){
-  if(err){
-    return err;
+  if (err) {
+    console.log(err);
+    return;
   }
   console.log(result);
-  translate(40, 0);
+  clear();
+  const padding = 20;
+  const rectWidth = (width - 2 * padding)/result.length
+  console.log({rectWidth, width, height})
```

Comment:
Suggestion:
* can re remove the `console.log()` here?

Reply:
Yes that was left in by accident